apps/wing/src/commands/pack.ts

```diff
@@ -120,6 +120,13 @@ export async function pack(options: PackageOptions = {}): Promise<string> {
       }
     }
 
+    // check package.json has `dependencies`
+    if (pkgJson.dependencies) {
```

Comment:
Add a check to make sure there are values in this key (an empty dep map is weird but fine)